GEOSagcm_GridComp/GEOSphysics_GridComp/GEOSsurface_GridComp/Utils/Raster/makebcs/CMakeLists.txt

```
@@ -47,3 +47,5 @@ ecbuild_add_executable (TARGET mk_runofftbl.x SOURCES mk_runofftbl.F90 LIBS MAPL
 ecbuild_add_executable (TARGET mkEASETilesParam.x SOURCES mkEASETilesParam.F90 LIBS MAPL ${this})
 
 install(PROGRAMS make_bcs clsm_plots.pro create_README.csh  DESTINATION bin)
+file(GLOB MAKE_BCS_PYTHON "./make_bcs*.py")
```

```suggestion
file(GLOB MAKE_BCS_PYTHON CONFIGURE_DEPENDS "./make_bcs*.py")
```

Ooh. @tclune is right. We want to add `CONFIGURE_DEPENDS` here. It's not a perfect solution, but without this option, if someone edited a python script, CMake would not know that the file changed. With the option, CMake will take the time to look and see if a file has changed.